web/org.eclipse.xtext.web.example.statemachine/src/org/eclipse/xtext/web/example/statemachine/GenerateStatemachine.mwe2

```diff
@@ -84,6 +84,8 @@ Workflow {
 				fragment = parser.antlr.XtextAntlrUiGeneratorFragment auto-inject {}
 			}
 			
+			fragment = parser.antlr.XtextAntlrGeneratorComparisonFragment auto-inject {}
```

Comment:
Since it is our only ready-to-use web-setup, it should still be idiomatic, I presume.